tests/test-sources/plugins/lsp/language-servers/nixd.nix

```diff
@@ -7,30 +7,22 @@
         enable = true;
 
         settings = {
-          eval = {
-            target = {
-              args = [
-                "foo"
-                "bar"
-              ];
-              installable = "";
-            };
-            depth = 0;
-            workers = 3;
-          };
+          nixpkgs.expr = ''
+            import (builtins.getFlake "/home/lyc/workspace/CS/OS/NixOS/flakes").inputs.nixpkgs { }
```

Comment:
Where does the referenced flake come from? Seems strange to include this in a test.

Reply:
Took the upstream documentation

Reply:
resolved then ?

Reply:
@MattSturgeon what do you think?

Reply:
I was reading it as a nix expression that'd actually get evaluated and was therefore very confused!

If it's just a string being assigned to an option then I guess it's fine.

Reply:
It is simply a string yes! `nixd` would evaluate that expression on LspAttach I believe